README.md

```diff
@@ -28,7 +28,7 @@ liboqs-dotnet
 
 This solution implements a .NET wrapper in C# for the C OQS library. It contains the following projects:
 
-- **dotnetOQS**: A C# class library targeting .NET Core 1.1, wrapping the master branch of the OQS library.
+- **dotnetOQS**: A C# class library targeting .NET Standard 1.2 and 2.0, wrapping the master branch of the OQS library.
```

Comment:
Why two version? We should list the minimum version we support; if it's 1.2, then I'd only leave 1.2. Or is the code different if you are targeting 2.0?

Reply:
The code is the same for those two target. However, the range of support is slightly different and the build size.

The main reason for having 2.0 is the reduced number of dependencies required when building an application. In that case, the app is smaller in size compared to 1.2. (Though the range of support is for newer .net core implementation)

Reply:
If there is an advantage to 2.0 then we should definitely keep it. Perhaps some rephrasing to say min version is 1.2 but 2.0 is preferred.

Reply:
I have done the README.md changes. Feel free to re-review it.
@christianpaquin